server/docs/currentISSLocation.md

```diff
@@ -1,53 +1,59 @@
-Current ISS Location 
-----
+## Current ISS Location
+
 ```
 Api will return the current location of the ISS in JSON data.
 ```
-**ENDPOINT** 
+
+**ENDPOINT**
+
 ```
 /api/currentISSLocation
 ```
+
 **Resource Information**
+
 ```
 Response Format: JSON
 ```
 
 #### URL Parameters
+
 No parameters.
-**METHOD** 
+**METHOD**
+
 ```
-Request type: 
+Request type:
 - GET
 ```
-Status Response
----
+
+## Status Response
+
 **Success Response:**
+
 ```javascript
 Content: {
   status: 200,
-  message: "Success getting ISS space station coordinates"m
+  message: "Success getting ISS space station coordinates",
   spaceLocationInformation: {
   lon: -30.2983,
-  lat: 4.6062
+  lat: 4.6062,
   timestamp: 1592250551,
   }
 }
 ```
+
 **Error Response:**
 No data returned from the API.
 
-```javascript
-  Code: 404 - NOT FOUND
-  Content: {status: 404, error: "No data available"} 
-```
 Request to the API failed.
+
 ```javascript
   Code: 502 BAD GATEWAY
   Content: {status: 404, error: "Received invalid response from API."}
 ```
-**Sample Call** 
+
+**Sample Call**
 
 ```javascript
-  fetch("/api/currentISSLocation")
+fetch("/api/currentISSLocation");
```

Comment:
Should have caught this earlier, but URLs should be all lowercase. Kebab case vs camel case if you need muiltiple words. i.e `/api/current-iss-location`